travis/.rubocop_rspec_base.yml

```diff
@@ -114,10 +114,61 @@ SpaceAroundEqualsInParameterDefault:
 StringLiterals:
   Enabled: false
 
+# By convention, RSpec examples use expect { ... } rather than expect do ... end
```

Comment:
IIRC it's not just by convention, there's a block priority to do with `do`/`end` that can affect us, which is negated by `{}`